k8s/aat/common-overlay/rd/kustomization.yaml

```diff
@@ -3,4 +3,6 @@ kind: Kustomization
 namespace: rd
 bases:
 - ../../../namespaces/rd
-- ../../../namespaces/rd/rd-caseworker-ref-api/rd-caseworker-ref-api.yaml
\ No newline at end of file
+- ../../../namespaces/rd/rd-caseworker-ref-api/rd-caseworker-ref-api.yaml
+patchesStrategicMerge:
+- ../../../namespaces/rd/rd-professional-api/aat.yaml
```

Comment:
why are u adding this ? Cannot see the patch crated . This shouldn't be needed if u are not overriding any behaviour